tools/build/build

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
This file needs to be executable. Run `git update-index --chmod=+x tools/build/build`